src/sdk/models/connections.ts

```diff
@@ -30,6 +30,8 @@ export interface PromptAttribute {
 	hint?: string;
 	required?: boolean;
 	value?: any;
+	help_text?: string;
+	show_default?: boolean;
```

Comment:
Where is `show_default` set on the backend?

Reply:
ShowDefault was initially added to the PromptAttribute class on the hotfix branch in https://github.com/portalarchitects/skysync-vnext/pull/3076. Once the hotfix is merged into develop, I will take this PR out of Draft status.